src/main/kotlin/io/sip3/commons/domain/Codec.kt

```diff
@@ -22,8 +22,8 @@ class Codec {
 
     var name: String = "UNDEFINED"
 
-    @JsonProperty("payload_type")
-    var payloadType: Byte = -1
+    @JsonProperty("payload_types")
+    var payloadTypes: List<Int> = listOf(-1)
```

Comment:
I think now when it's an object we can say `lateinit var payloadTypes: List<Int>`